templates/block-block-js.mustache

```diff
@@ -1,33 +1,70 @@
 ( function( wp ) {
+	var registerBlockType = wp.blocks.registerBlockType;
 	var el = wp.element.createElement;
 	var __ = wp.i18n.__;
 
-	// Visit https://wordpress.org/gutenberg/handbook/block-api/ to learn about Block API
-	wp.blocks.registerBlockType( '{{namespace}}/{{slug}}', {
+	/**
+	 * Every block starts by registering a new block type definition.
+	 * The function takes two arguments, a block name and a block configuration object.
+	 * @see https://wordpress.org/gutenberg/handbook/block-api/
+	 */
+	registerBlockType( '{{namespace}}/{{slug}}', {
+		/**
+		 * This is the display title for your block, which can be translated with `i18n` functions.
+		 * The block inserter will show this name.
+		 */
 		title: __( '{{title_ucfirst}}', '{{textdomain}}' ),
```

Comment:
IIRC the `domain` is not handled yet and I wonder if the i18n module is ready for third-party usage

Reply:
Should I remove `domain` part or leave it as it is? I think it doesn't harm to have 2nd param :)

Reply:
Yes, I should remove the `domain` part from JS file. Related documentation in Gutenberg: https://github.com/WordPress/gutenberg/tree/master/i18n#usage.

> Note that you will not need to specify domain for the strings.